package.json

```diff
@@ -33,7 +33,9 @@
     "hammerjs": "2.0.8",
     "helmet": "3.6.1",
     "highcharts": "5.0.12",
+    "history": "4.6.3",
     "keythereum": "1.0.0",
+    "lodash": "^4.17.4",
```

Comment:
Point 1 — yes.  I intentionally made sure our build would not include the entirety of the lib, just the methods we need, that’s why you’ll see import statements like: `import isEqual from ‘lodash/isEqual’;`. It’s got some super useful helper methods that are just faster to import than re-build.  

Point 2 — Good spot, missed that semver caret.